### PR TITLE
[v0.3] DE-NAS readme update

### PR DESCRIPTION
**Pull request by Jian-Zhang**
Updated readme for DE-NAS

### CODE REVIEW

File: e2eAIOK/DeNas/README.md [hunk 1]
```diff
@@ -9,34 +32,35 @@ git submodule update --init --recursive
...
 ```
 
-# Run quick try for ASR model
+#### Trial for ASR model
 
 ```
 python -u search.py --domain asr --conf ../../conf/denas/asr/e2eaiok_denas_asr.conf
-```
\ No newline at end of file
+```
```
Comment (tianyil1) [inline e2eAIOK/DeNas/README.md:66]: Maybe we can add the training script following the search process.

Comment (xuechendi) [reply e2eAIOK/DeNas/README.md:66]: @tianyil1, let's adding training process after demo and performance refreshed

**Review by xuechendi — APPROVED** [review]
looks good

File: e2eAIOK/DeNas/README.md (hunk 1 above)
Comment (tianyil1) [reply e2eAIOK/DeNas/README.md:66]: Got it. Thx.

**Review by tianyil1 — APPROVED** [review]

**Pull request merged**